.changeset/perfect-seas-push.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/versions": patch
+---
+
+Command `fuels version` not working
```

Comment:
```suggestion
Fix for `fuels version` throwing when the user does not have `forc` or `fuel-core` installed.
```

Nit, more descriptive.

Reply:
This comes from this [conversation](https://github.com/FuelLabs/fuels-ts/pull/1873#discussion_r1528379611), but I agree it feels a bit off. 🤔 

cc @nedsalk

Reply:
Definitely off, but happy to leave the resolution to #1872

Reply:
We can decouple the PR title from the changeset description, still have @arboleya's [proposed approach](https://github.com/FuelLabs/fuels-ts/issues/1028#issuecomment-1999722353) for PR and changelog structure, and continue with the changeset descriptions as we have previously done, knowing that they don't matter that much in the end?

Reply:
I am trying to unify the `changelog entry` with their respective `commit message`. Pairing them for easy reference would make sense, leaving all the details in the PR description.

Could it be more straightforward if the changeset was identical to the PR title?

This:
```
fix: command `fuels version` not working
```

Instead of this:
```
Command `fuels version` not working
```

Reply:
That looks okay as well. At least for me, though, this rule of PR_TITLE==CHANGESET_DESCRIPTION will be a source of small frustration whenever I forget to update the changeset description, but I can live with it.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1883/commits/178d5da5ff4da8540ac81a13d1822e743ad86a0f